src/lib/components/dropdown/Dropdown.component.js

```diff
@@ -8,23 +8,45 @@ import {
   ButtonText
 } from "../button/Button.component";
 import * as defaultTheme from "../../style/theme";
-import { getThemePropSelector } from "../../utils";
+import { getThemePropSelector } from '../../utils';
+import { getPositionDropdownMenu } from './utils';
 
 export type Item = {
   label: string,
   name?: string,
   selected?: boolean,
-  onClick: any => void
+  onClick?: any => void,
+  submenuIcon?: Node,
+  submenuItems?: Array<Item>
```

Comment:
As shown in the design, the documentation and support item also have another external link icon.
Maybe we miss another `iconExternal` prop?

Reply:
done